src/components/UserSelector.tsx

```diff
@@ -73,10 +64,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(setAuthor(user));
```

Comment:
Using an anchor (`<a>`) tag for actions that do not lead to navigation is not recommended. Anchor tags should be used for navigation purposes, and buttons should be used for actions. In this case, it would be more semantically correct to use a `<button>` element with `type="button"`.